core/lib/services/writables.ts

```diff
@@ -23,3 +23,17 @@ export const typeFunction: WritableWithDefaultOptions<(...args: any[]) => any> =
 	normalizeValue: testToNormalizeValue(isFunction),
 	equal: Object.is,
 };
+
+export const typeArray: WritableWithDefaultOptions<any[]> = {
+	normalizeValue: testToNormalizeValue(isArray),
+	equal: (a: any[], b: any[]) => {
```

Comment:
```suggestion
	equal: (a, b) => {
```
(the type of equal is already implied by `WritableWithDefaultOptions<any[]>` line 27)

Reply:
it is, however, the IDE highlights it in red saying that it as implicit type